WooCommerce/WooCommerceUITests/Tests/LoginTests.swift

```diff
@@ -13,7 +13,7 @@ final class LoginTests: XCTestCase {
 
     // Login with Store Address and log out.
     func test_site_address_login_logout() {
-        let prologue = PrologueScreen().selectSiteAddress()
+        let prologue = try! PrologueScreen().selectSiteAddress()
```

Comment:
Is there a way to make this fail with a prettier error message of what happened? 😅

Reply:
Hey @shiki! I'm glad you caught this `try!` 💪 

It's totally _not_ something I would endorse in the final implementation. Here, I took a shortcut just to get the code to compile.

Extra details from [the matching PR on WordPress iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/16648#discussion_r647155154)

> I force tried (`try!`) here because I didn't want to keep traversing the call graph. In the "final" implementation, I'd like not to have any `try!`. The reason I'm saying this is because XCTest knows what to do when a `try` `throws` and reports it as a test failure instead of a crash.
>
> _Shameless plug: I wrote of such an instance [today](https://mokacoding.com/blog/how-to-test-async-await-code-in-swift/)._

Reply:
> Here, I took a shortcut just to get the code to compile.

Ah make sense! Sorry for the drive-by comment. 😅 I was just curious. This PR looks interesting!